server/activityItem/__tests__/fetch.test.ts

```diff
@@ -111,7 +112,9 @@ const models = modelize`
 
 setup(beforeAll, models.resolve);
 
-teardown(afterAll);
+teardown(afterAll, async () => {
+	await finishDeferredTasks();
+});
```

Comment:
This is in some ways the solution to the memory leak problems in the tests I ran into before.

Some tests use methods which create or update ActivityItems, which are deferred by that util. Sometimes, they do not finish before the test is over, so they stick around and Jest does not clean them up properly, leading to leaked memory.

How I solved this is by just forcefully closing the connection to the DB every time (this is in the `teardown` function), but that sometimes leads to (non-test-ruining) errors of `DB was closed before transaction was completed`, because the deferred tasks are still running. 
To solve that and to prevent possible future errors, I just wait until all deferred tasks are done before closing the connection.

Now that I think of it, it's probably better to just do this in the `teardown` function in general rather than adding it where it causes problems huh